dmutils/__init__.py

```diff
@@ -4,4 +4,4 @@
 import flask_featureflags  # noqa
 
 
-__version__ = '31.2.0'
+__version__ = '32.0.0'
```

Comment:
If we're just adding a feature, this isn't a major (breaking) change, so could this be `31.3.0`?

Reply:
We're changing an existing filter now to add ` BST` or ` GMT` on the end. It's probably borderline whether this is a breaking change - anyone else have a strong opinion?

Reply:
It's a change of design, but won't break any existing code as far as I can see - you will be able to pull this change in to apps without needing to change any application code (tests that check date strings don't count), so definitely not a breaking change to me.

Reply:
In fact, I'd probably argue that this is only a patch change - it's not a new feature, it's an adjustment of an existing feature, so I'd probably go `31.2.1`.

Reply:
Count me convinced 😸 @brenetic do you want to do the honours?